src/index.html

```diff
@@ -7,8 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logoImg"
+      >
```

Comment:
`alt` attribute should describe the image content ([checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md))

```suggestion
        <img
            src="./images/logo.png"
            alt="Moyo logo"
            class="logo-img"
      >
```